api/app/Models/Timezones.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace App\Models;
+
+use Illuminate\Support\Collection;
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * Class Timezones
+ * @package App\Models
+ *
+ * Note this model does not have an associated database table as it is an abstract
+ * data model with generated token data.
+ */
+class Timezones extends Model
```

Comment:
Why does it etends Model then?
Can we make it TimezonesFactory with single static call?